contracts/base/Lenders.sol

```diff
@@ -122,22 +122,18 @@ contract Lenders is Base, LendersInterface {
         isInitialized()
         returns (uint256)
     {
-        uint256 amountToWithdraw = amount;
-        uint256 withdrawableInterest = accruedInterest[recipient].totalNotWithdrawn;
-
-        if (withdrawableInterest == 0 || amountToWithdraw == 0) return 0;
-
-        if (withdrawableInterest < amountToWithdraw) {
-            amountToWithdraw = withdrawableInterest;
-        }
-
-        accruedInterest[recipient].totalNotWithdrawn = withdrawableInterest.sub(
-            amountToWithdraw
+        require(
```

Comment:
Would it also make sense to `require(...)` the `amount` to be non-zero?

Reply:
Fixed!